pages/schedule/index.tsx

```diff
@@ -272,9 +272,18 @@ export default function Calendar(props: { scheduleCard: ScheduleEvent[] }) {
                 <Toolbar />
                 <DateNavigator />
                 <TodayButton />
-                <GroupingState grouping={grouping} groupByDate={() => true} />
-                <IntegratedGrouping />
-                <GroupingPanel />
+                {
+                  // since tracks are computed from entries, only show grouping if there are any tracks
+                  uniqueTracks.size > 0 ? (
```

Comment:
your code is breaking: https://hackportal-git-fix-191-acmutd.vercel.app/schedule
`Application Error: A client-side exception has occurred`

Reply:
So I messed around with this for a lil bit on my local machine and uhhhh for some reason replacing the react fragment tags with an array of components fixes the issue (no clue why):

![image](https://user-images.githubusercontent.com/37679458/221396003-e4632ef6-f341-4c02-a8dd-7d283c5fe031.png)

Reply:
Whoops I'm dumb. Should be fixed by 608bc0638d760aaac2e2f0e8dfa877915ee2dfec. Not sure how this slipped through.

Reply:
> So I messed around with this for a lil bit on my local machine and uhhhh for some reason replacing the react fragment tags with an array of components fixes the issue (no clue why):
> 
> ![image](https://user-images.githubusercontent.com/37679458/221396003-e4632ef6-f341-4c02-a8dd-7d283c5fe031.png)

Oh I didn't see this. 608bc0638d760aaac2e2f0e8dfa877915ee2dfec did not work after all, so I'll do this instead.

Reply:
Okay React is still complaining when I do that. The only way I can get it to work is if I render each in separate template braces.

![image](https://user-images.githubusercontent.com/33424649/221396693-8ec9f445-8e68-4ad4-9571-6c3fd5791bd7.png)

It's not horribly ugly but it feels kinda wrong. Does anyone know a better way to render groups of components that have to be rendered in a specific order? I don't know why the nested block syntax isn't working.

Reply:
The issue is that, for some god-forsaken reason, React Scheduler components rely on other components being rendered before them. I don't know why that's a pattern that exists, but it does, so the cleanest solution I can think of is to render each component in separate braces. This makes me very sad.

Reply:
React Scheduler is and has been a garbage library, we've just got to work around its BS until we finish the refactor and can move to a different library or create something custom.